search.html

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Wondering if we will eventually drop this html file in replace of a search bar drop down with suggestions? Challenging?

Reply:
possibly ~ definitely doable.